src/utils/tsc.cpp

```diff
@@ -20,12 +20,14 @@ extern "C" {
 namespace memgraph::utils {
 uint64_t ReadTSC() { return rdtsc(); }
 
-std::optional<double> GetTSCFrequency() {
+bool IsAvailableTSC() {
   // init is only needed for fetching frequency
-  static auto result = std::invoke([] { return rdtsc_init(); });
-  return result == 0 ? std::optional{rdtsc_get_tsc_hz()} : std::nullopt;
+  static bool available = [] { return rdtsc_init() == 0; }();  // iile
```

Comment:
`iile` in comment is misprint or some abbreviation?

Reply:
Immediately invoked lambda expression